beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -89,7 +89,7 @@ pub const MAX_WORK_EVENT_QUEUE_LEN: usize = 16_384;
 const MAX_IDLE_QUEUE_LEN: usize = 16_384;
 
 /// The maximum size of the channel for re-processing work events.
-const MAX_SCHEDULED_WORK_QUEUE_LEN: usize = 16_384;
+const MAX_SCHEDULED_WORK_QUEUE_LEN: usize = (0.75 * MAX_WORK_EVENT_QUEUE_LEN as f64) as usize;
```

Comment:
I'm generally suspicious of float arithmetic, even though in this case it would be insignificant if there were representation issues.

```suggestion
const MAX_SCHEDULED_WORK_QUEUE_LEN: usize = 3 * MAX_WORK_EVENT_QUEUE_LEN / 4;
```